src/EmployerAan/SFA.DAS.EmployerAan.Api.UnitTests/Controllers/CalendarsControllerTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.EmployerAan.Api.Controllers;
+using SFA.DAS.EmployerAan.Application.Calendars.Queries.GetCalendars;
+using SFA.DAS.EmployerAan.Entities;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.EmployerAan.Api.UnitTests.Controllers;
+public class CalendarsControllerTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task And_MediatorCommandSuccessful_Then_ReturnOk(
+        List<Calendar> response,
+        [Frozen] Mock<IMediator> mockMediator,
+        CancellationToken cancellationToken)
+    {
+        mockMediator.Setup(m => m.Send(It.IsAny<GetCalendarsQuery>(), cancellationToken)).ReturnsAsync(response);
+        var controller = new CalendarsController(mockMediator.Object);
```

Comment:
This could have been initialised in the parameters as well using [Greedy] attribute.

Reply:
Done